### ISSUE TITLE
Cannot create Content Packs

### ISSUE THREAD

**Issue by edmundoa** [Graylog2/graylog2-server#5381]
Using the current master I cannot create Content Packs or Content Pack revisions. Clicking on "Create" or "Create and download" at the end of the wizard throws an error in the browser console:

```
TypeError: this.state.contentPack.toObject is not a function
	_onSave CreateContentPackPage.jsx:58
	_onSave (index):974
```

And no new Content Packs are created. Funnily enough, I can download the Content Pack and upload it to create the new Content Pack.

## Steps to Reproduce (for bugs)
1. Go to the Content Packs page
2. Click on _Create a Content Pack_ and follow the wizard
3. The last step is to Create or Create and download the Content Pack, clicking on Create does nothing, but throws an error in the browser's console. Clicking on Create and Download starts the file download, but no new Content Packs are created, and there's also an error in the browser's console.

## Your Environment
* Graylog Version: Graylog 3.0.0-alpha.5-SNAPSHOT
* Browser version: Firefox Developer Edition 65.0b1

**Comment by bernd** [comment]
I can reproduce that. 

@kmerz This small change seems to fix it:

```diff
diff --git graylog2-web-interface/src/pages/CreateContentPackPage.jsx graylog2-web-interface/src/pages/CreateContentPackPage.jsx
index 28a61abf6..860113b5e 100644
--- graylog2-web-interface/src/pages/CreateContentPackPage.jsx
+++ graylog2-web-interface/src/pages/CreateContentPackPage.jsx
@@ -55,7 +55,7 @@ const CreateContentPackPage = createReactClass({
   },
 
   _onSave() {
-    ContentPacksActions.create.triggerPromise(this.state.contentPack.toObject())
+    ContentPacksActions.create.triggerPromise(this.state.contentPack.toJSON())
       .then(
         () => {
           UserNotification.success('Content pack imported successfully', 'Success!');
```

**Comment by bernd** [comment]
This has been fixed in #5357.
